packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_def.hpp

```diff
@@ -106,6 +106,13 @@ namespace FROSch {
             this->ParameterList_->set("RCP(Phi)", Phi_);
         }
 
+        // Store current Coarse Matrix in ParameterList_
+        if ( this->ParameterList_->get("Store Coarse Matrix",false) ) {
+            FROSCH_NOTIFICATION("FROSch::CoarseOperator",this->Verbose_,"Storing current Coarse Matrix in Parameterlist.");
+            this->ParameterList_->set("RCP(Coarse Matrix)", CoarseMatrix_);
```

Comment:
Does this work correctly when you store a matrix in the parameterlist? I always thought it has to be serializable, may be the matrix is serializable?

Reply:
We also store parallel multivectors (coordinate list, null space) in the parameter list. Therefore, I was assuming that this works for matrices as well. 

Thank you for the comment. I will keep an eye on this.